### ISSUE TITLE
Most recent 0.7 broken with Django deployment

### ISSUE THREAD

**Issue by benjiec** [webcomponents/webcomponentsjs#320]
In Django templates, you can have webcomponents.somehash.js -- this breaks webcomponents.js when it loads. Error:

Uncaught TypeError: Cannot read property 'getAttribute' of null

This is because the script tag with webcomponents.js does not exist.

**Comment by benjiec** [comment]
Unfortunately, looks like it's broken in 0.6.2 as well. Working in 0.6.1. Frustratingly, due to various issues with bower, it's very difficult to get the deployment to stick to 0.6.1 over 0.6.2.

**Comment by azakus** [comment]
Yeah, that was a busted tag. I've killed that tag, and tagged 0.6.1 as 0.6.3 so bower will pick up the correct build.
Sorry for the confusion!
